meta/main.yml

```diff
@@ -15,9 +20,11 @@ galaxy_info:
   min_ansible_version: "2.10"
   namespace: cisagov
   platforms:
-    - name: Amazon Linux
-      versions:
-        - "2023"
+    # Amazon Linux 2023 does not yet support Docker Compose. See the package
```

Comment:
Per my other comment please update this entry if testing succeeds.

Reply:
See [1ab5bfa](https://github.com/cisagov/ansible-role-admiral/pull/3/commits/1ab5bfa60235a1fd9824b5031176e2397df0851f).